navigation/routes.ts

```diff
@@ -1,4 +1,4 @@
 export enum Routes {
-  CURRENCIES_LIST = 'currencies-list',
+  CURRENCIES_LIST = 'list',
   ADD_CURRENCY= 'add-currency'
```

Comment:
[minor] Route constants should have the same value as the name, e.g "CURRENCIES_LIST = 'CURRENCIES_LIST"

Reply:
Also, the correct form of the subject is "Currency list"